packages/cards/graphql/schema-types.js

```diff
@@ -0,0 +1,56 @@
+module.exports = `
+
+type Card {
+  id: ID!
+  payload: JSON!
+  group: CardGroup!
+  user: User
+  documents: DocumentConnection!
+  match: Boolean
+  totalMatches: Int!
```

Comment:
```suggestion
  totalMatches: Int!
  statement: Comment
```

I want to query a preview of that comment